### DISCUSSION TITLE
Ubuntu端的问题

### DISCUSSION THREAD

**Discussion by houbing2006** [nextai-translator/nextai-translator#451] [Q&A]
安装后，Ubuntu22只有图标，无法启动应用

**Comment by 1915419156** [comment]
![image](https://user-images.githubusercontent.com/66114894/227752367-44fa770d-b891-42a3-ba24-7ef1798eeb14.png)
我的也是安装完成后点图标没反应

**Reply by 1915419156** [reply]
我的是Ubuntu20.04版本

**Reply by fgc346** [reply]
![image](https://user-images.githubusercontent.com/22359933/227821562-0d7c75b3-4839-40e5-9386-55f403368a22.png)
我的是ubuntu18.04版本，显示缺少glibc_2.29和glibc_2.28。查了一下ubuntu18.04不支持glibc_2.29，升级有风险。

**Reply by hongyi-zhao** [reply]
无法运行，将停留在下面的步骤：
```shell
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.10
Release:	22.10
Codename:	kinetic
$ open-ai-translator 
Error: permission denied for this operation, plese try as super user
Thumb window does not exist

```

**Comment by maltoze** [comment]
v0.0.36已修复此问题

**Reply by hongyi-zhao** [reply]
已确认此修改正常工作：

![image](https://user-images.githubusercontent.com/11155854/228394541-0fe68673-c4be-4425-ac69-e9102856ca33.png)

另外，在上面的主窗口上，应该也加上 `Hide `， `Show` 按钮，比较方便。
